hsuforum_actions_lib.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+// required for LIKE functionality move to hsuforum.
+
+
+/**
+* Forum Actions
+*
+* @package   hsuforum_actions
+* @copyright 2014 Moodle Pty Ltd (http://moodle.com)
+* @author    Mikhail Janowski <mikhail@getsmarter.co.za>
+* @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+*/
+
+// Add javascript to the advanced forum pages
+// if (isset($_SERVER['REQUEST_URI']) && strpos($_SERVER['REQUEST_URI'], 'mod/hsuforum/discuss.php')) {
+//    global $PAGE;
+//    // $PAGE->requires->js_call_amd('local_hsuforum_actions/hsuforum_actions', 'init');
+// }
+
+// This function is included to force moodle to run the local lib.php on every page.
+function local_hsuforum_actions_extend_navigation($param) {}
```

Comment:
Lets see how we can remove this. Not sure how much you looked into this.

Reply:
not much as yet I want to.